src/pages/AdminDashboard/AdminInvestment.tsx

```
@@ -1,9 +1,176 @@
 
+import { Tab, Tabs, TabsBody, TabsHeader } from "@material-tailwind/react";
+import { useState } from "react";
+import { Table } from "../../components/Table";
+import { FaRegArrowAltCircleRight } from "react-icons/fa";
+// import { FaRegArrowAltCircleRight } from "react-icons/fa";
+
 const AdminInvestment = () => {
+   const [activeTab, setActiveTab] = useState("investment1");
+   const [selectedItem,setSelectedItem] = useState({})
```

ESLint

[error] 'selectedItem' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)